src/Uno.UI/UI/Xaml/Controls/Grid/Grid.Properties.cs

```diff
@@ -131,7 +131,7 @@ private void OnCornerRadiusPropertyChanged(CornerRadius oldValue, CornerRadius n
 		#endregion
 
 		#region Row Property
-		[GeneratedDependencyProperty(DefaultValue = 0, AttachedBackingFieldOwner = typeof(UIElement), Attached = true, ChangedCallbackName = nameof(OnGenericPropertyChanged))]
+		[GeneratedDependencyProperty(DefaultValue = 0, Options = FrameworkPropertyMetadataOptions.AffectsMeasure | FrameworkPropertyMetadataOptions.AffectsArrange, AttachedBackingFieldOwner = typeof(UIElement), Attached = true)]
```

Comment:
Are we invoking InvalidateMeasure somewhere in the callback that we should be removing?

Reply:
You can scroll horizontally and see I did that ;-)